### ISSUE TITLE
Ways to pass custom `mysqldump` options?

### ISSUE THREAD

**Issue by dabico** [databacker/mysql-backup#346]
I did not manage to find any information in the documentation (both the `dump --help` and in the `configuration.md` file), but is there a way I can pass custom flags to the `dump` command to include objects like `triggers` and `events` in my dump? For example, I use the following for `mysqldump`:

```
[mysqldump]
quick
events
triggers
routines
single-transaction
max-allowed-packet=1G
```

**Comment by deitch** [comment]
That is because mysql-backup (v1.0.0+ versions) doesn't actually use `mysqldump`. It is not a script, but a single binary, that incorporates mysql functionality to create the dump.

If there is functionality missing, definitely open to adding it; that is how it has grown to this point. And Pull Requests obviously are much appreciated.

**Comment by dabico** [comment]
Thank you for the fast answer! 😄

I've never programmed in Go, but I'll take a glance at the code and give it a try. Always wanted to learn the language too.

**Comment by deitch** [comment]
Sure thing. Feel free to ask questions here.

**Comment by deitch** [comment]
Nope, I do not. I wouldn't mind having them, but that level of DBA (it really is deeper than that) is not my expertise. It is a general "how to optimize backups" question. I know people who do that, but I do not.
